### PR TITLE
Expand text on stewardship goals

### PR DESCRIPTION
**Pull request by Ainali**
Adding analogy and clarify that we don't integrate

### CODE REVIEW

File: activities/codebase-stewardship/goals.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ Prospective users can trust there will be packages when they need to update thei
 Future contributors can trust that there will be someone to receive their pull request after the original maintainers have moved on.
 
 We provide this sustainability in the first place by creating strong communities of users and developers that can take ownership of the product.
+
+We don't give a man a fish. We don't even assume he wants to learn fishing. We help him figure out what nourishment he needs and then help him along on that path, regardless if it is becoming a fisherman or a farmer.
```
Comment (bvhme) [inline activities/codebase-stewardship/goals.md:27]: Putting multiple sentences on one line makes reordering content, reviewing subsequent changes and navigating its history significantly harder. Putting them on multiple lines still keeps them in the same paragraph. I recommend one line per paragraph.

Comment (bvhme) [reply activities/codebase-stewardship/goals.md:27]: Our previous discussion: https://github.com/publiccodenet/about/pull/613.

Comment (Ainali) [reply activities/codebase-stewardship/goals.md:27]: Should we write that into a manual of style, or in the contributing file (which by the way uses multiple sentences on the same line)?

Comment (bvhme) [reply activities/codebase-stewardship/goals.md:27]: Yeah, I know, I've been planning a big 'find and replace' but it's been hard to get it done. :/

That's also why I've approved this, I think it is ready to go. I'll just keep pointing at it for my own `git blame` happyness. :)

**Pull request merged**